library/pom.xml

```diff
@@ -366,6 +366,37 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-common</artifactId>
+      <version>${hadoop.version}</version>
+      <type>test-jar</type>
+      <scope>test</scope>
+      <exclusions>
+        <exclusion>
```

Comment:
I could not get what you mean by exclude all dependencies. I tried with 
`
<exclusion>
           <groupId>*</groupId>
           <artifactId>*</artifactId>
         </exclusion>
`
This gives compilation issues.